components/ImageUploader.tsx

```diff
@@ -200,10 +201,10 @@ export default function ImageUploader({ target, id, buttonMsg, handleAvatarChang
               </div>
               <div className="mt-5 sm:mt-4 sm:flex sm:flex-row-reverse">
                 <button type="button" className="btn btn-primary" onClick={showCroppedImage}>
-                  Save
+                  <FormattedMessage id="save" defaultMessage="Save" />
                 </button>
                 <button onClick={closeImageUploadModal} type="button" className="btn btn-white mr-2">
-                  Cancel
+                  <FormattedMessage id="cancel" defaultMessage="Cancel" />
```

Comment:
we should abstract this into a shorter component 

something i've seen people do is


```

<T>Hello</T>

```

Reply:
this is fixed

Reply:
oh, not the way I was thinking off. you've just renamed the import. technically makes it shorter but I was thinking of a different implementation:

In a previous project that I worked on we had a wrapper function `<T>Hello</T>` that would create an id by lowercasing and removing spaces and special characters.

you write: `<T>Hello</T>` and it generates: `<FormattedMessage id="hello" defaultMessage="Hello" />`

the speed of writing strings is the most important part of i18n. an annoyingly long API will slow down development.

Reply:
also, you're still directly importing react-intl in every component vs. abstracting it into components/T.jsx